tests/unit/src/kakarot/accounts/eoa/test_library.py

```diff
@@ -45,17 +45,31 @@ async def test_execute_should_make_all_calls_and_return_concat_results(
                 )
             ]
             calldata += tx
-            expected_result += [
-                int(transaction["to"], 16),
-                transaction["value"],
-                transaction["gas"],
-                len(
-                    bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                ),  # calldata_len
-                *list(
-                    bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                ),  # calldata
-            ]
+            # Execute contract bytecode
```

Comment:
@ClementWalter  kinda hijacked your test here.
I didn't think that this deserved its own one, as we're still only testing whether transactions are decoded and forwarded to kakarot correctly.